client/reader/stream/post-lifecycle.jsx

```diff
@@ -54,7 +54,7 @@ class PostLifecycle extends Component {
 					<PostPlaceholder />
 				</Fragment>
 			);
-		} else if ( post._state === 'error' ) {
```

Comment:
I couldn't find anywhere that `post._state` is ever set to `'error'`. This appears to be [left over from our flux days](https://github.com/Automattic/wp-calypso/blob/cd6589a30e7e8c7093e32f08101026cbba342d3e/client/lib/feed-post-store/index.js#L239).